src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -125,10 +130,23 @@ const CustomCollection = ({
     </>
   )
 
-  // Component and modal for Settings to pass as
-  // Custom Collection header
-  const customCollectionSettings = (
+  const customCollectionHeader = (
     <>
+      <EditableInput
+        childRef={inputRef}
+        text={customTitle}
+        placeholder={'Add a title for this collection'}
+        onSave={handleEditCollection}>
+        <input
+          id="editableCollectionTitle"
+          type="text"
+          name="title"
+          value={customTitle}
+          ref={inputRef}
+          placeholder={'Add a title for this collection'}
+          onChange={(e) => setCustomTitle(e.target.value)}
+        />{' '}
+      </EditableInput>
```

Comment:
Q: Could we render the `<input>` element directly inside of `EditableInput` instead of passing in children? Then I don't think we'd need to be passing ref objects around. This component doesn't seem to use `inputRef` other than to pass it back into `EditableInput`